app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -132,9 +126,11 @@ fun SpotListScreen(
                   val distance = TurfMeasurement.distance(referencePoint, parking.location.center)
                   SpotCard(navigationActions, parkingViewModel, parking, distance)
                   Log.d("ListScreen", "Filtered parking spots: $filteredParkingSpots")
-                  if (filteredParkingSpots.indexOf(parking) == filteredParkingSpots.size - 1) {
-                    parkingViewModel.getKClosestParkings(
-                        referencePoint, numberOfNewParkings + filteredParkingSpots.size)
+                  if (filteredParkingSpots.indexOf(parking) == filteredParkingSpots.size - 1 &&
+                      radius.doubleValue < 1000) {
+                    // This incremental solution could be improved to be dynamic and with a limit
+                    radius.doubleValue += 100
```

Comment:
Try to use constants for the radius step and max radius. I know you'll work on the infinite scroll list soon so this is not a matter for this PR, but make sure to to it.

Reply:
Noted, I'll change this when fixing the bug with 0 parking displayed